tests/integration/cmd_devfile_push_test.go

```diff
@@ -13,18 +13,20 @@ import (
 )
 
 var _ = Describe("odo devfile push command tests", func() {
-	var project string
+	var namespace string
 	var context string
 	var currentWorkingDirectory string
 
 	var sourcePath = "/projects"
 
+	// TODO: all oc commands in all devfile related test should get replaced by kubectl
+	// TODO: to goal is not to use "oc"
```

Comment:
Please create a github issue so that we won't loose eye on it

Reply:
#2752